lib/README.md

```diff
@@ -342,6 +342,7 @@ bluecove-gpl-2.1.1-SNAPSHOT.jar
 ##### PlantUML
 - plantuml.jar
     was from plantuml.org, now from https://github.com/plantuml/plantuml.git
+    pulled June 1, 2021 and modified with PR#1028
```

Comment:
PR #1028 is Table menus. It doesn't look right.

Reply:
That's PR#1028 on PlantUML: https://github.com/plantuml/plantuml/pull/1028 Will clarify